config/services/storinfo/service.json

```diff
@@ -1,6 +1,6 @@
 {
 	"params": {
 		"networks": [ "manta", "admin" ],
-		"ram": 256
+		"ram": 1024
```

Comment:
FWIW in my COAL (granted idle), my storinfo is happily at only 90MB of RAM:

```
[root@headnode (coal) ~]# zonememstat -a
                                 ZONE                                             ALIAS  RSS(MB) CAP(MB)    NOVER  POUT(MB) SWAP%
...
 31dea065-d727-475b-80aa-856d4630e431            storinfo.coalregion.joyent.us-31dea065       90    256        0         0 24.353
```

Do we know that it needs to be bigger for *COAL*?  I guess @khangngu had some experience... at "lab" size?
If it is a pain to come up with a number now, we can always come back and trim down *coal* sizes separately.

Reply:
Checking the labs which have up to 6 storage zones, storinfo still stay below 256. I've walked back on the change for coal (and lab).